client/src/components/PaperSearchBar/PaperSearchBar-container.tsx

```diff
@@ -10,7 +10,6 @@ const paperSearch = async (input: string): Promise<Paper[]> => {
     const { data } = await axios.get<PaperResponse[]>(`/api/search/${input}`);
     return data.map(constructPaperFromResponse);
   } catch (err) {
-    console.error(err);
```

Comment:
This wasn't useful? Or leaked sensitive info?

Reply:
Neither; just a runtime warning and I never checked it 🤷